tahoe_idp/backend.py

```diff
@@ -116,8 +117,8 @@ def _build_user_details(self, jwt_payload, auth0_user):
             "fullname": fullname,
             "first_name": first_name,
             "last_name": last_name,
-            "auth0_is_organization_admin": is_organization_admin(metadata_role),
-            "auth0_is_organization_staff": is_organization_staff(metadata_role),
+            "tahoe_idp_is_organization_admin": is_organization_admin(metadata_role),
+            "tahoe_idp_is_organization_staff": is_organization_staff(metadata_role),
```

Comment:
I'll take care of those changes in another PR. Please undo changes in this file.